MARKDOWN-GUIDE.md

```diff
@@ -0,0 +1,162 @@
+# Markdown enabled fields
+Markdown syntax can be used in the following fields:
+- /document/acknowledgments[]/summary
+- /document/distribution/text
+- /document/notes[]/text
+- /document/publisher/issuing_authority
+- /document/references[]/summary
+- /document/tracking/revision_history[]/summary
+- /product_tree/product_groups[]/summary
+- /vulnerabilities[]/acknowledgments[]/summary
+- /vulnerabilities[]/involvements[]/summary
+- /vulnerabilities[]/notes[]/text
+- /vulnerabilities[]/references[]/summary
+- /vulnerabilities[]/remediations[]/details
+- /vulnerabilities[]/remediations[]/entitlements[]
+- /vulnerabilities[]/remediations[]/restart_required/details
+- /vulnerabilities[]/threats[]/details
```

Comment:
Please put these lines as code block or quote each single line. (The slash must be removed - but if I do it in the suggestion it will fail...)
```suggestion
\```
/document/acknowledgments[]/summary
/document/distribution/text
/document/notes[]/text
/document/publisher/issuing_authority
/document/references[]/summary
/document/tracking/revision_history[]/summary
/product_tree/product_groups[]/summary
/vulnerabilities[]/acknowledgments[]/summary
/vulnerabilities[]/involvements[]/summary
/vulnerabilities[]/notes[]/text
/vulnerabilities[]/references[]/summary
/vulnerabilities[]/remediations[]/details
/vulnerabilities[]/remediations[]/entitlements[]
/vulnerabilities[]/remediations[]/restart_required/details
/vulnerabilities[]/threats[]/details
```